File: explorer/src/main/java/com/duongame/manager/FontManager.java
package com.duongame.manager;

import android.content.Context;
import android.graphics.Typeface;

/**
 * Created by js296 on 2017-07-08.
 */

public class FontManager {
    static Typeface nanumMyeongjo;

    public static Typeface getTypeFaceNanumMeyongjo(Context context) {
        if (nanumMyeongjo == null) {
            nanumMyeongjo = Typeface.createFromAsset(context.getAssets(), "fonts/NanumMyeongjo.ttf");
        }
        return nanumMyeongjo;
    }
}

File: Explorer/src/main/java/com/duongame/manager/FontManager.kt
package com.duongame.manager

import android.content.Context
import android.graphics.Typeface

/**
 * Created by js296 on 2017-07-08.
 */
object FontManager {
    private var nanumMyeongjo: Typeface? = null

    fun getTypeFaceNanumMeyongjo(context: Context): Typeface? {
        if (nanumMyeongjo == null) {
            nanumMyeongjo = Typeface.createFromAsset(context.assets, "fonts/NanumMyeongjo.ttf")
        }
        return nanumMyeongjo
    }
}